src/components/atoms/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC, DetailedHTMLProps, AnchorHTMLAttributes } from "react";
+import "./LinkButton.scss";
+
+export interface LinkButtonProps
+  extends DetailedHTMLProps<
+    AnchorHTMLAttributes<HTMLAnchorElement>,
+    HTMLAnchorElement
+  > {
+  href: string;
+}
+
+export const LinkButton: FC<LinkButtonProps> = ({ children, href }) => (
+  <a href={href} target="_blank" rel="noreferrer" className="button-link">
+    {children}
+  </a>
+);
```

Comment:
This should use the `externalUrlAdditionalProps` const from `utils/url`.

Additionally, you say that it accepts all `DetailedHTMLProps` in `LinkButtonProps` yet you don't actually apply them to the `a` tag. In addition to this, we need to make sure that if `className` is passed, it doesn't overwrite our default styles here.

Our style is to use `React.FC` rather than just `FC`.

```suggestion
export const LinkButton: React.FC<LinkButtonProps> = ({ children, href, className, ...extraProps }) => {
  const linkClasses = classNames("button-link", className)
  
  return (
    <a href={href} className={linkClasses} {...externalUrlAdditionalProps, ...extraProps}>
      {children}
    </a>
  );
}
```

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/3632c4de26b887043c8c61f1daad943747517382

Reply:
Looks like you changed the ordering of the props added to the `<a` from what I suggested. Why is that?

Reply:
@oleksii-front This still seems to be outstanding/requires answering?

Reply:
NB